src/server/serverSetup.js

```diff
@@ -196,3 +229,19 @@ sjrk.storyTelling.server.resolveJSONFile = function (jsonFilePath) {
     var file = kettle.resolvers.file(jsonFilePath);
     return JSON.parse(file);
 };
+
+sjrk.storyTelling.server.getCustomThemeFolder = function (theme, themeFolder) {
+    var folder = ".";
+
+    if (theme) {
+        var resolvedFolder = fluid.stringTemplate(themeFolder, { theme: theme });
+        // ensure the folder exists, otherwise return an error
+        if (fs.existsSync(resolvedFolder)) {
+            folder = resolvedFolder;
+        } else {
+            folder = undefined;
```

Comment:
When the custom theme folder is not found, starting the server will throw a fatal error. It would be more user friendly to report a proper error message using fluid.fail() and quit.